<ui version="4.0" >
 <class>config</class>
 <widget class="QWidget" name="config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Sample &amp;Rate</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="1" >
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QSlider" name="IntervalHorizontalSlider" >
              <property name="minimum" >
               <number>20</number>
              </property>
              <property name="maximum" >
               <number>1000</number>
              </property>
              <property name="singleStep" >
               <number>20</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KIntNumInput" name="intervalEdit" >
              <property name="value" >
               <number>500</number>
              </property>
              <property name="minimum" >
               <number>20</number>
              </property>
              <property name="maximum" >
               <number>1000</number>
              </property>
              <property name="suffix" >
               <string> msec</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Scaling</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7" >
          <item row="1" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Paging:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSlider" name="horizontalSlider_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="KIntNumInput" name="pageScaleEdit" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&amp;Swapping:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSlider" name="horizontalSlider_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="KIntNumInput" name="swapScaleEdit" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&amp;Context switch:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QSlider" name="horizontalSlider_4" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2" >
           <widget class="KIntNumInput" name="ctxScaleEdit" />
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="autoScaleBox" >
            <property name="text" >
             <string>&amp;Automatic</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <spacer name="horizontalSpacer_9" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="HorizontalBarsCheckBox" >
         <property name="text" >
          <string>Horizontal Bars</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>C&amp;olors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>CPU</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Kernel:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="CpuKernel_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>User:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="CpuUser_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Nice:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="CpuNice_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>IOWait:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="CpuIowait_kcolorbutton" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Memory</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Kernel:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="MemKernel_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>Used:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="MemUsed_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>Buffers:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="MemBuffers_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>Cached:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="MemCached_kcolorbutton" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Swap</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>Swap:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="SwapSwap_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_7" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>Background:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KColorButton" name="SwapBackground_kcolorbutton" />
            </item>
            <item>
             <spacer name="horizontalSpacer_8" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <attribute name="title" >
       <string>&amp;Interaction</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="MouseEventsGroupBox" >
         <property name="title" >
          <string>Mouse Events</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_14" >
            <property name="text" >
             <string>Left button:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="LeftMouseActionComboBox" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimumContentsLength" >
             <number>20</number>
            </property>
            <item>
             <property name="text" >
              <string>Is Ignored</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Switches Mode</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Pops Up Menu</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Starts</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Middle button:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="MiddleMouseActionComboBox" >
            <item>
             <property name="text" >
              <string>Is Ignored</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Switches Mode</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Pops Up Menu</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Starts</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_16" >
            <property name="text" >
             <string>Right button:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QComboBox" name="RightMouseActionComboBox" >
            <item>
             <property name="text" >
              <string>Is Ignored</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Switches Mode</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Pops Up Menu</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Starts</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="KUrlRequester" name="RightMouseActionKurlrequester" />
          </item>
          <item row="1" column="2" >
           <widget class="KUrlRequester" name="MiddleMouseActionKurlrequester" />
          </item>
          <item row="0" column="2" >
           <widget class="KUrlRequester" name="LeftMouseActionKurlrequester" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>162</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>swapScaleEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>246</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ctxScaleEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntervalHorizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>intervalEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>110</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctxScaleEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>188</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntervalHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>174</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel" >
     <x>123</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>pageScaleEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>182</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel" >
     <x>223</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>swapScaleEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>187</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel" >
     <x>287</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageScaleEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>311</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel" >
     <x>181</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
